.github/workflows/e2e.yml

```diff
@@ -0,0 +1,77 @@
+name: e2e
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    services:
+      docker:
+        image: docker:dind
+
+    steps:
+      - name: Check machine OS
+        run: |
+          echo "Machine OS: ${{ runner.os }}"
+
+      - name: Check Docker version
+        run: |
+          docker version
+
+      - name: Install Docker Compose
+        uses: ndeloof/install-compose-action@v0.0.1
+        with:
+          version: v2.5.1
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 6.32.9
+          run_install: |
+            - recursive: true
+            - args: [--frozen-lockfile, --strict-peer-dependencies]
+
+      - name: Create proj folder and add keys and genesis.json
```

Comment:
Remove unnecessary steps

Reply:
This is required for running L1 and L2 as per steps in docker